### ISSUE TITLE
(Feature request) Cancelling the `@` trigger when we really just want to use the @ character

### ISSUE THREAD

**Issue by diyoyo** [zadam/trilium#4692]
### Describe feature

Very often, I want to type word with an @ that are not related to note-linking.
Just because I like the emphasis it gives to a word, or because I'm implicitly writing that I'm referring to an instagram account or whatsoever.

### Additional Information

Currently, the only way to skip the 'linking note' automation, is to use the right-arrow key. But that's never enough, any time I edit the note, the 'linking note' automation comes back and interfere with my current flow.

At the end of the day, this is almost a bug report rather than a feature request, I'd say.

**Comment by diyoyo** [comment]
> It probably doesn't help much, but typing any charachter before the @ symbol prevents the note linking dropdown from showing up. so you could do something like `.@yourt_text` if that's not too much of a distraction.

Well, that is too much of a distraction, since I have cases where the previous character must be a space, in a long sentence ending with @something . Yes, I'm being picky here, sorry 🙈
I tried adding special code quotes:  `@helloworld` like in markdown , and I'm not bothered by the menu while I am typing. But... the menu appears after I type the closing quote :(

**Comment by X-Bruce-Y** [comment]
Hi I'm thinking about 2 ways. If it does not distract you much, you can *finish typing the whole paragraph after @ and finally ESC*. Otherwise you may try always use AT or some rare combinations you like to indicate @ and *write a script to replace AT with @ hourly or manually through launch bar*.

**Comment by zadam** [comment]
Hi, I implemented a workaround which should deactivate the "mention UI" on escape.

It's kind of a hack by inserting a zero width space which acts as a disallowed character for the autocomplete.

Unfortunately the implementation in CKEditor is regex based and given we allowed spaces for searching in the autocomplete, we need some other way to terminate the autocomplete.

This will be released in the next patch release.
